packages/flutter/lib/src/widgets/two_dimensional_viewport.dart

```diff
@@ -1088,6 +1134,8 @@ abstract class RenderTwoDimensionalViewport extends RenderBox implements RenderA
   /// it if it already exists.
```

Comment:
This method can have more detail doc string. Maybe add some comment around when this method should be call (in layoutChildSequence I think?) and the childvicinity must be what is visible on the screen because this method will insert the child into active children list

Reply:
The children aren't required to be visible since they could be in the cache extent and are ok as active children there. I will add more docs though.

Reply:
Ah thank you!